draft-ietf-quic-recovery.md

```diff
@@ -1480,8 +1480,8 @@ GetLossTimeAndSpace():
 GetPtoTimeAndSpace():
   duration = (smoothed_rtt + max(4 * rttvar, kGranularity))
       * (2 ^ pto_count)
-  // Arm PTO from now when there are no inflight packets.
-  if (no in-flight packets):
+  // Arm PTO from now when there are no ACK-eliciting packets.
+  if (no ack-eliciting packets in flight):
```

Comment:
Ack-eliciting packets are defined, so I'd use that.

```suggestion
  if (no ack-eliciting packets):
```

Reply:
I prefer this as-is, as it's explicit that we're talking about ACK-eliciting packets that have been transmitted but neither acknowledged nor declared lost. The exact wording is also already used elsewhere, e.g. line 1529.

I appreciate that "in flight" is being a bit overloaded here, so switching all three cases over to a different wording might be nice, but it's not immediately obvious to me what that might be.

Reply:
If you want something positive the condition is "all ack-eliciting packets sent by this endpoint have either been acknowledged or marked lost".  So maybe "all ack-eliciting packets accounted for".

Reply:
"accounted for" seems less precise/precedented than "in flight" to me, at least.

Reply:
Thanks for clarifying why you wanted to state "in flight" as well as ack-eliciting.  I think the extra clarity is good.  However, now the comment doesn't quite match up.

I agree the "accounted for" is less precise than "in flight" in this case.

Reply:
Tweaked the comment to match.

Reply:
This method doesn't actually arm the PTO, and the whole method is about PTO, so maybe it'd be best to remove the comment?

Alternatively, you could change it to:
"Base the PTO from now when no ACK eliciting packets are in flight to avoid it being set too early."